terraform_templates/iam_template.py

```diff
@@ -0,0 +1,23 @@
+from typing import Mapping
+
+
+def terraform_iam_template(config: Mapping):
+    dict_template = {
+        "terraform": {
+            "required_providers": {
+                "aws": {
+                    "source": "hashicorp/aws",
+                    "version": "~> 3.00"
```

Comment:
I would pin this version to be exact.  Terraform tends to be messy in between versions in my experience.

Reply:
Pinned to 3.00